spec/controllers/verify_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe VerifyController do
+  describe '#show' do
+    subject(:response) { get :show }
+
+    before do
+      routes.draw { get 'show' => 'verify#show' }
+    end
+
+    after { Rails.application.reload_routes! }
+
```

Comment:
I don't think we need these, now that the routes are drawn in `routes.rb` unconditionally?

```suggestion
```

Reply:
> I don't think we need these, now that the routes are drawn in `routes.rb` unconditionally?

Good catch! Forgot to remove these.